src/templates/components/qaSection/index.tsx

```diff
@@ -11,9 +11,16 @@ export function QaSection({
   displayAccordion,
 }: FormattedQaSection | FormattedQaGroup) {
   const setHeaderh3 = header ? true : false
+  // Prepare id for use by va-on-this-page component to identify the QaSection
+  const headerId = header
+    ? header
+        .toLowerCase()
+        .replace(/[^a-z0-9\s]/g, '')
+        .replace(/\s+/g, '-')
+    : ''
```

Comment:
Can you pull this into a helper function that we can leverage in other components too?

Reply:
Great idea! I'll pull this into a reusable helper function today.

Reply:
Going to prepare this helper function led to me finding an already existing `slugifyString` to use for this id prep 🎊  so I went ahead and used that instead of recreating